scripts/collector/postgres/sql/base/database_details.sql

```diff
@@ -22,6 +22,7 @@ with db as (
     pg_encoding_to_char(db.encoding) as character_set_encoding,
     pg_database_size(db.datname) as total_disk_size_bytes
   from pg_database db
+  where datname = current_database() 
```

Comment:
There are several other places where we'll likely need this as well.  Also, there are copies of these queries in `src/dma` that will need to be updated.